src/fragments/lib/auth/ios/credential_handling/10_credential_handling.mdx

```diff
@@ -0,0 +1,5 @@
+## Credential handling
+
+Amplify Swift library securely handles credentials and user identity information. Customers do not need to store, refresh, or delete credentials themselves. Instead, Amplify Swift library handles these operations using platform capabilities such as [`ASWebAuthenticationSession`](https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession) for web-based OAuth flows, and [Keychain Services](https://developer.apple.com/documentation/security/keychain_services/) for storing and retrieving sensitive information. Only the host app (or defined app group) can access values that Amplify Swift library stores, and Amplify Swift library also clears credentials if it detects a configuration change such as a new Cognito UserPool or Cognito Identity Pool.
```

Comment:
I was just reading the style guide - got another nit: "Customers do not need to store, refresh, etc.." Can we change that to "You do not need...."